zyte_common_items/pipelines.py

```diff
@@ -46,22 +44,15 @@ def parse_product(self, response: DummyResponse, product: Product):
     """
 
     def __init__(self):
-        warn(
```

Comment:
We have warning in the ae module. So, This is a redundant warning. I guess, we can remove it!